modules/apps/frontend-js/frontend-js-svg4everybody-web/build.gradle

```diff
@@ -0,0 +1,44 @@
+import com.liferay.gradle.util.copy.StripPathSegmentsAction
+
+configurations {
+	coreJSBundle
+}
```

Comment:
Is this necessary? I copy/pasted it from the ie-compatability module

Reply:
No, this is unnecessary. This defines the line below as a configuration

> coreJSBundle group: "org.webjars.npm", name: "core-js-bundle", transitive: false, version: "3.1.4"

That means that we're downloading that thingie from maven and then you can use it.

In the IE module we download core-js that way and then extract its contents. Definitely not needed here.